bindings/pydeck/PUBLISH.md

```diff
@@ -47,6 +47,10 @@ twine upload dist/*
 
 8) Verify again the pydeck installed from the main pypi works in the environment above.
 
+9) Update the contents of `./conda.recipe` in this directory, using the diff between the current meta.yml and `conda skeleton pydeck`. Follow the steps documented [here](https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-packages/#uploading-conda-packages)
```

Comment:
Can this be automated?

Reply:
Addressed in the pydeck makefile: https://github.com/uber/deck.gl/pull/4213/files#diff-988e12fde0ff5641422002e60e8be3f0